requirements/edx/base.txt

```diff
@@ -116,7 +116,7 @@ edx-django-oauth2-provider==1.3.3
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
 edx-drf-extensions==1.5.2
-edx-enterprise==0.70.6
+edx-enterprise==0.70.7
```

Comment:
In future we just need to specify the enterprise version in `base.ini` and `make upgrade` command will automatically update the related requirements files.
Read the detail here: https://github.com/edx/edx-platform/pull/18534#issuecomment-403726276